src/Model/User.php

```diff
@@ -244,6 +244,11 @@ function ( $cap ) {
 
 					return ! empty( $user_locale ) ? $user_locale : null;
 				},
+				'isToolbarVisible'         => function () {
+					$toolbar_preference_meta = get_user_meta( $this->data->ID, 'show_admin_bar_front', true );
+
+					return $toolbar_preference_meta === 'true' ? true : false;
```

Comment:
I think the word `toobar` is too generic here. The official name is [Admin Bar](https://developer.wordpress.org/reference/functions/show_admin_bar/), though I can hear an argument for `adminToolbar` to be more self-documenting for non WP-first users.

Also wonder if `is{$}Visible` is the correct phrasing since that has a frontend connotation that is only true when querying for `viewer` and not in the context of any other `User` reference in the schema. I think `has{$}` or `shouldShow{$}` better indicates the value in the field (former is an existing pattern e.g. `hasArchive`, but the latter is more descriptive for users with a WP background since it closer matches the WP API)

Reply:
I think the word `toobar` is too generic here. The official name is [Admin Bar](https://developer.wordpress.org/reference/functions/show_admin_bar/), though I hear an argument for `adminToolbar` to be more self-documenting for non WP-first users.

Reply:
@justlevine FWIW From my understanding admin bar was the original name, as reflected in the code style, and Toolbar is how it's now referenced within the UI.

<img width="801" alt="user profile settings page" src="https://user-images.githubusercontent.com/6676674/225628182-fcb81e30-9a53-42d1-87d1-64902704cbd0.png">

Given the context of these changes I would agree that admin_bar should be used here 👍

Reply:
@josephfusco and it's actually been called Toolbar in the UX since 3.1, whoops 😬

In that case I guess I'd recommend `{has|shouldShow}Toolbar` if it's only old fogies like me that still think of it as the "Admin Bar", or `shouldShowAdminToolbar` to help with self documentation if the description is not enough. I would _not_ recommend just `{prefix}AdminBar` since it's _only_ fully self descriptive for people who are familiar with internal WP APIs.

Reply:
I think in terms of self documentation and user understanding, `shouldShowAdminToolbar` may be the best bet here. Thoughts? @josephfusco @justlevine

Reply:
Just renamed the property to `shouldShowAdminToolbar` 👍